controllers/storageclass/storageclass_controller.go

```diff
@@ -192,6 +187,34 @@ func (r *StorageClassWatcher) getFlashSystemClusterByStorageClass(sc *storagev1.
 	return foundCluster, fmt.Errorf(msg)
 }
 
+func (r *StorageClassWatcher) getSecret(sc *storagev1.StorageClass) (corev1.Secret, error) {
+	secret := &corev1.Secret{}
+	secretName, ok := sc.Parameters[util.DefaultSecretNameKey]
+	if !ok {
+		secretName, ok = sc.Parameters[util.ProvisionerSecretNameKey]
+		if !ok {
+			return *secret, fmt.Errorf("cannot find/identify secret name in StorageClass")
+		}
+	}
+	secretNamespace, ok := sc.Parameters[util.DefaultSecretNamespaceKey]
+	if !ok {
```

Comment:
it means that one can create a secret using default secret name, but provisioner secret namespace.
is it supported? for creating a valid PVC